sw/device/silicon_creator/manuf/lib/personalize.c

```diff
@@ -256,23 +150,11 @@ static status_t flash_keymgr_secret_seed_write(
  * @return OK_STATUS on success.
  */
 OT_WARN_UNUSED_RESULT
-static status_t otp_partition_secret2_configure(
-    const dif_otp_ctrl_t *otp_ctrl,
-    wrapped_rma_unlock_token_t *rma_unlock_token) {
+static status_t otp_partition_secret2_configure(const dif_otp_ctrl_t *otp_ctrl,
+                                                const lc_token_hash_t *hash) {
```

Comment:
```suggestion
                                                const lc_token_hash_t *rma_unlock_token_hash) {
```
nit: update "@param[out]" above: this is no longer an output